src/Entity/IdeasWorkshop/Question.php

```diff
@@ -0,0 +1,157 @@
+<?php
+
+namespace AppBundle\Entity\IdeasWorkshop;
+
+use Algolia\AlgoliaSearchBundle\Mapping\Annotation as Algolia;
+use AppBundle\Entity\EntityPublishableTrait;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+use Gedmo\Mapping\Annotation as Gedmo;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * @ORM\Table(name="ideas_workshop_question")
+ * @ORM\Entity
+ *
+ * @UniqueEntity("name")
+ *
+ * @Algolia\Index(autoIndex=false)
+ */
+class Question
+{
+    use EntityPublishableTrait;
+
+    /**
+     * @var int
+     *
+     * @ORM\Column(type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     */
+    private $id;
+
+    /**
+     * @var Guideline
+     *
+     * @ORM\ManyToOne(targetEntity="Guideline", inversedBy="questions")
+     */
+    private $guideline;
+
+    /**
+     * @ORM\Column
+     */
+    private $placeholder;
+
+    /**
+     * @ORM\OneToMany(targetEntity="Answer", mappedBy="question")
+     */
+    private $answers;
```

Comment:
I think we can avoid the bidirectional relation here, one answer must have a relation with question, but one question can have to much answers for all ideas of all adherents